Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,71 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="Styles.css" </head> <body>
+
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+
+    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+        intelligent.</h2>
+
+
+    <p1>By Carl Zimmer </p1>
+
+    <p2>Nov. 30, 2018</p2>
+
+    <figure>
+        <img src="Images/merlin_147494628_a788760f-4209-40db-a0be-1305c3a661a6-superJumbo.jpg" alt="octopus swimming under the sea">
+        <figcaption>An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchersare
+            uncertain how the trait evolved.</figcaption>
+        <img <figure>
```

Comment:
2 fixes here:
1. You have a stray `<img`
2. You need to close the `figure` element
```suggestion
        </figure>
```